mozetl/constants.py

```diff
@@ -0,0 +1,7 @@
+# Restrict to a whitelist of search_source's to avoid double counting while
+# we test our new search_count telemetry developed in:
+# https://bugzilla.mozilla.org/show_bug.cgi?id=1367554
+# https://bugzilla.mozilla.org/show_bug.cgi?id=1368089
+source_whitelist = [
```

Comment:
nit: [recommended PEP8 style](https://www.python.org/dev/peps/pep-0008/#id48) is to use uppercase for constants. Also, if this file is going to be for general-purpose "constants", we should call this var something like `search_source_whitelist` for clarity.